DigraphyApi/Services/OrderService.cs

```diff
@@ -14,4 +15,18 @@ public async Task<Result<ICollection<OrderDto>>> GetOrdersAsync(int? collectionI
             : await orderRepository.GetOrdersByCollectionIdAsync(collectionId.Value, isVerified);
         return mapper.Map<List<OrderDto>>(orders);
     }
+
+    public async Task<Result<ICollection<TitleDto>>> GetOrderAsync(int orderId)
+    {
+        var order = await orderRepository.GetOrderAsync(orderId);
+        
+        if (order == null)
+        {
+            return Errors.OrderNotFound(orderId);
+        }
+
+        var titles = order.Titles.OrderBy(t => t.ReleasedAtUrc).ToList();
```

Comment:
The order of titles can be found using the `OrderTitle.Index` property... Also, you'll need to `Include` this property in the `Repository` for this to be populated. You can sort it like:

```cs
.Include(o => o.Titles.OrderBy(t => ...))
```

Although the index is in the join table, so you may need to include the `OrderTitles`... Potentially you'll need to add the `Title` entity into the `OrderTitle` class like so:

```cs
 public class OrderTitle
{
    public int TitleId { get; set; }
    public int OrderId { get; set; }
    public int Index { get; set; }
    
    public Title Title { get; set; }
    public Order Order { get; set; }
}
```

I'm honestly not quite sure the best way to do it, but it would be good if we could do it in SQL.